iio/iio_app/iio_app.c

```diff
@@ -195,7 +195,7 @@ int32_t iio_app_run(struct iio_app_device *devices, int32_t len)
 	if (status < 0)
 		return status;
 #endif
-
+#ifndef LINUX_PLATFORM
```

Comment:
this commit looks good but please rephrase that commit title to something more meaningful